app/src/main/res/layout/component_tabstray2.xml

```diff
@@ -105,7 +105,7 @@
             android:layout_height="match_parent"
             android:contentDescription="@string/tabs_header_synced_tabs_title"
             android:foregroundTint="@color/photonWhite"
-            android:icon="@drawable/ic_synced_tabs" />
+            android:icon="@drawable/ic_synced" />
```

Comment:
Please undo renaming `ic_synced_tabs`. Our icon naming should follow the naming defined by UX Icon design system. This way we can easily link back the figma icon to our icon naming.

Reply:
how can I find the responding icon? 

the file name [used](https://www.figma.com/file/am9EJGg6ow29PXWkc9jcOH/MR2-(2021)?node-id=1311%3A244184) in the synced screen is `Synchronize tabs`.